### ISSUE TITLE
where is  config.json?

### ISSUE THREAD

**Issue by ljljlj02** [bharathikannan1311/AI-for-Medicine-Specialization-deeplearning.ai#5]
Hello, in "AI for Medical Diagnosis-- > Week 3--- Brain Tumor Auto-segmentation for Magnetic Resonance Imaging (MRI)-->C1M3_Assignment.py ", Line 1200-1206, contains the following paragraph:

# run this cell if you didn't run the training cell in section 4.1
base_dir = HOME_DIR + "processed/"
with open(base_dir + "config.json") as json_file:
    config = json.load(json_file)
# Get generators for training and validation sets
train_generator = util.VolumeDataGenerator(config["train"], base_dir + "train/", batch_size=3, dim=(160, 160, 16), verbose=0)
valid_generator = util.VolumeDataGenerator(config["valid"], base_dir + "valid/", batch_size=3, dim=(160, 160, 16), verbose=0)``

Where did you get the config.json file? 
Any answers would be greatly appreciated.

**Comment by nbharathik** [comment]
Hello, config.json file should be included in the Coursera assignment environment. I have uploaded only the assignment files, ungraded lab and some supporting files. If you wish to get this file, Download your entire directory in the Coursera Environment.
